sopel/plugins/rules.py

```diff
@@ -484,15 +493,11 @@ def is_limited(
             # not even started, so not limited
             return False
 
-        # detect if we just started something or if it ended
-        last_time = self.started_at
-        if self.ended_at and self.started_at < self.ended_at:
```

Comment:
This was used to check that the rule's execution ended. Because rule can be running in a thread, you have no guaranty that a second execution is not trying to do stuff at the same time, so the `last_return_value` must not apply unless it has actually ended first.

Reply:
Thanks, I've restored that check in fb774ba